Firestore/Example/Tests/Util/FSTIntegrationTestCase.mm

```diff
@@ -212,14 +215,18 @@ - (void)readerAndWriterOnDocumentRef:(void (^)(NSString *path,
 }
 
 - (FIRDocumentSnapshot *)readDocumentForRef:(FIRDocumentReference *)ref {
+  return [self readDocumentForRef:ref getOptions:[FIRGetOptions fromDefault]];
+}
+
+- (FIRDocumentSnapshot *)readDocumentForRef:(FIRDocumentReference *)ref getOptions:(FIRGetOptions *)getOptions {
```

Comment:
You should use this in all of your tests! :)

Reply:
Doesn't work when I expect an error, but otherwise, yeah, I really should. Done.